cip-XXXX-Synchronizer-Migration-to-Canton-3-3/cip-XXXX-Synchronizer-Migration-to-Canton-3-3.md

```diff
@@ -0,0 +1,99 @@
+- Many of the choices in the Token Standard are dependent on ledger time. Expressing these constraints using getTime limits the maximal delay between preparing a transaction and submitting it for execution to one minute. Canton 3.3 introduces new primitives for asserting constraints on ledger time, which remove that artificially tight one minute bound, and instead capture the actual dependency of the submission time and ledger time. These new primitives are used in the `Amulet` implementation. 
+- Composing Canton Network applications is best done via interfaces to minimize the required coordination for rolling out smart contract upgrades of dependencies. In Canton 3.2, this did not work well, as the version of Daml packages were chosen using purely local information about the submitting validator node instead of considering the packages vetted by the counter-participants involved in the transaction. Canton 3.3 introduces support for vetting-state-aware package selection to lower the required coordination for rolling out smart contract upgrades in many cases to zero.
+- Application developers need stable and easily accessible APIs to build on. Canton 3.3 takes a major step forward in stabilizing the developer facing APIs of the Canton participant node by making all calls available in the gRPC Ledger API also accessible via HTTP JSON with OpenAPI schema definitions being provided for easy discoverability. Thus allowing developers to freely pick between gRPC or HTTP JSON depending on what suits them best.
+- The Canton 3.3 release also introduces experimental Daml support for verifying cryptographic signatures. Thereby making it easier to build Daml workflows that bridge between Canton Network and other chains.
+
+## Rationale 
+The Canton 3.3 release is available for upgrade.  It enables new features in Splice 0.4.0, as well as new capabilities to be leveraged by Canton Network application developers and partners. A preview of the Canton 3.3 release notes is available [here](https://blog.digitalasset.com/developers/release-notes/canton-daml-3.3-preview).
+
+The Canton 3.3 release introduces the following new capabilities:
+- Easier application upgrades:  [Smart contract upgrading (SCU)](https://docs.daml.com/upgrade/smart-contract-upgrades.html#smart-contract-upgrade) that allows Daml models (packages in DAR files) to be updated transparently, provided guidelines in making application changes are followed.  
+- Support for the Canton Network Token Standard. SCU allows the CNTS to evolve without breaking changes.                                                                      
+- Enhancements for the interactive submission service (e.g., Time boundaries functions).  The interactive submission service is enabled by default.
```

Comment:
it was already enabled by default in splice0so that change isn't meaningful.
"Enhancements" is also quite ambiguous. I'd make it about the concrete change of introducing time boundaries

Reply:
Changed to "Added time boundaries functions for the interactive submission service. "